diff --git a/Makefile.vc b/Makefile.vc
index 67d0cf3d..b181a499 100755
--- a/Makefile.vc
+++ b/Makefile.vc
@@ -69,7 +69,6 @@ FLINKFLAGS =
 #               objects and executables.
 #
 all :
-	copy pythonapi\src\calcephpy.pyx.vc pythonapi\src\calcephpy.pyx
 	cd src && nmake /f Makefile.vc  CCFLAGS="$(CFLAGS)" CCC="$(CC)" FFC="$(FC)" FFFLAGS="$(FFLAGS)" FFLINKFLAGS="$(FLINKFLAGS)"
 	cd examples && nmake /f Makefile.vc  CCFLAGS="$(CFLAGS)"  CCC="$(CC)" FFC="$(FC)" FFFLAGS="$(FFLAGS)" FFLINKFLAGS="$(FLINKFLAGS)"
 	cd tests && nmake /f Makefile.vc  CCFLAGS="$(CFLAGS)"  CCC="$(CC)" FFC="$(FC)" FFFLAGS="$(FFLAGS)" FFLINKFLAGS="$(FLINKFLAGS)"
